runtime/lua/vim/secure.lua

```diff
@@ -86,21 +113,75 @@ function M.read(path)
     trust[fullpath] = hash
   end
 
+  write_trust(trust)
+
+  return contents
+end
+
+--- Update the trust status of file at path in the trust database at
+--- $XDG_STATE_HOME/nvim/trust.
+---
+--- @param path (string) Path to a file to update status for.
+--- @param mode (string) One of the following:
+---   - 'allow': Add file to trust database and set it as trusted
+---   - 'deny': Add file to trust database and set it as denied
+---   - 'forget': Remove file from trust database
+--- @return (boolean, string|nil) success, errmsg: true and nil if the operation was successful,
+---   otherwise false and an error message.
+function M.trust(path, mode)
+  vim.validate({ path = { path, 's' } })
+  vim.validate({ allow = { mode, 's' } })
```

Comment:
```suggestion
  vim.validate({
    path = { path, 's' },
    mode = { mode, function(v)
      return v == 'allow' or v == 'deny' or v == 'forget'
    end, '"allow", "deny" or "forget"' },
  })
```

Reply:
I did this slightly differently based on @gpanders suggestion before I saw your comment, I hope it's okay :) I see you're using `"` to wrap allowed values, but somewhere else I found single quotes used and that's what I used as well. I assume that'd mean there is no set in stone convention for that, but maybe we lean towards one or the other? Should I change these to `"`?

related commit: dc8040d5ecd62fb3235ac730d2e744727c0c6bc1

Reply:
As long as the linter is not complaining, it's probably fine either way.